handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,169 @@
+
+**Keywords**: `code host`, `cloning`, `syncing`, `token`, `GitHub`, `GitLab`, `Bitbucket`, `Phabricator`, `Gerrit`, `repository`, `project`, `Perforce`, `src-expose`
+
+Any questions about code host connections and repository syncing should be routed to the [Core application team](../engineering/core-application/index.md).
+
+Note that this section applies to backend connections with code hosts, such as repository cloning and syncing. Questions about [frontend/UI integrations with code hosts](#browser-extensions-and-code-host-native-integrations) (e.g., getting code intelligence inside of a code host) should be rounted to the [Web team](../engineering/web/index.md).
+
+### Repository permissions
+
+**Keywords**: `permissions`, `ACLs`, `access`, `authorization`, `authz`
+
+Any questions about repository permissions should be routed to the [Core application team](../engineering/core-application/index.md).
+
+### User authentication (SSO)
+
+**Keywords**: `authentication`, `authn`, `SSO`, `SAML`, `OAuth`, `auth proxy`, `OpenIDConnect`, `OIDC`
+
+Any questions about user authentication should be routed to the [Core application team](../engineering/core-application/index.md).
+
+### Code intelligence
+
+**Keywords**: `code intelligence`, `precise`, `navigation`, `LSIF`, `language server`/`LSP`, `go to definition`, `hover`, `find references`, `ctags`, any programming language names (e.g. `Go`, `Java`, `Python`, `COBOL`, etc.)
+
+Any questions about code intelligence and navigation should be routed to the [Code Intelligence team](../engineering/code-intelligence/index.md).
+
+### Search
+
+**Keywords**: `search`, `indexed search`, `indexing`, `diff search`, `symbols`, `keyword`, `filter`, `scope`, `version context`, `repogroup`, `saved search`
+
+Any questions about search should be routed to the [Search team](../engineering/search/index.md).
+
+### Browser extension and code host native integrations
+
+**Keywords**: `browser extension`, `native integration`, `chrome`, `firefox`, `safari`, `Phabricator`, `Bitbucket`, `GitHub`, `GitLab`
+
+Any questions about the browser extension or code host native integrations should be routed to the [Web team](../engineering/web/index.md).
+
+### Sourcegraph extensions
+
+**Keywords**: `extensions`, `plugins`, `blame`, `git-extras`, `Codecov`
+
+Any questions about Sourcegraph extensions should be routed to the [Web team](../engineering/web/index.md).
+
+### Campaigns
+
+**Keywords**: `campaigns`, `large scale code changes`, `refactoring`, `src-cli`
+
+Any questions about campaigns should be routed to the [Campaigns team](../engineering/campaigns/index.md).
+
+
+## Step 4: File a Github issue
+
+It can seem duplicative, but it's just as important when we need help as it is when we file a defect to file a Github issue. This allows us to understand how we are doing toward our goal of being a self-sustaining team, as well as trends over time that impact the customer experience and our team's experience.
```

Comment:
Is it useful to add clarification that it also helps other teams plan workloads for on call folks appropriately? I don't know if it matters/we want to include it, but that is I think the reason on the eng side.

Reply:
Yes! Updated.